### PR TITLE
issue #2 - login implement

### PR DESCRIPTION
**Pull request by chibicibianca**

### CODE REVIEW

File: backend/src/callbacks/login.ts [hunk 1]
```diff
@@ -10,15 +10,47 @@ export const initializeLoginCallback = (app: express.Application) => {
     authorizeCallbackHook(req, res);
 
     const loggedInUser: Auth0User = req.body.user;
+    console.log('loggedInUser: ', JSON.stringify(loggedInUser, null, 2));
```
Comment (sensus261) [inline backend/src/callbacks/login.ts:13]: Delete this please

File: backend/src/callbacks/login.ts [hunk 2]
```diff
@@ -10,15 +10,47 @@ export const initializeLoginCallback = (app: express.Application) => {
...
+      const identityAlreadyAdded = !!userIdentities.find((databaseIdentity) => {
+        if (databaseIdentity.connection === incomingIdentity.connection) {
+          return true;
+        }
+
+        return false;
+      });
+
+      if (identityAlreadyAdded === false) {
+        prisma.entUserIdentity.create({
```
Comment (sensus261) [inline backend/src/callbacks/login.ts:38]: Please `await` this condition

**Review by sensus261 — COMMENTED** [review]
.

File: backend/src/callbacks/login.ts (hunk 1 above)
Comment (chibicibianca) [reply backend/src/callbacks/login.ts:13]: nu

**Pull request merged**